javascripts/main.js

```diff
@@ -35,6 +35,28 @@
   });
 
   $(function() {
+    $(document).on("mouseenter focus mouseleave", '.label a', function(){
```

Comment:
This was hard to discover, could this be just the entire cell - aka `.projects tbody`?